assets/js/script.js

```diff
@@ -93,4 +93,12 @@ $(document).ready(function() {
 		$('#warn-invalid-link').attr({ 'href': href, 'target': target });
 		$('#warn-invalid-modal').modal();
 	});
+
+	// Comment this code to debug the site in the console
+    console.clear()
+    if (window.console || 'console' in window) {
+        setTimeout(console.clear())
+        // Logo ASCII displayed in the console
+        setTimeout(console.log("                                                                                \n                                   ????                 ((((                    \n                                ????????  (((        ((((((((                   \n                            ???????????? (((((((  ((((((((((((                  \n                        ????????#??####?(((((((((((((((((((((((                 \n                    ####################((((((((((((((((((((((((                \n                ##########################################(((((((               \n            ##        (###       ####??   ##            #####  (((              \n        ######   #########   ###   ????   #######   ########/   ###             \n    ##########   #########   #???   ???   #######   ########     ###            \n (((((((((((((        ((((   ????   ???   #######   #######   #   ###((#        \n    ((((((((((   ((((((???   ?(    ????   ???????   ??????   ???   ???#####     \n        ((((((   (((??????      ???????   ???????   ?????           ???#######  \n           (((        #???   ??????????   ???????   ????   ???????   ???#####   \n             ??????????????&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&??????       \n          ???? ??????  ?? && & &  &  &&? & & ?& && &  && & ???? ??   ?????      \n       ???????????????&&&&&&&&&&&&&&&&&&&&&&&&&&???????????????????????????     \n                       ???????????????(((                                       \n                      ?????? ((((((((((((                                       \n                                 ((((((((                                       \n                                    ((((((                                      \n                                        ((                                      "));
```

Comment:
What's the point of `setTimeout`?
The console log function is executed in-place. 
So your code has a type error: `setTimeout` takes `void` as argument.

Also for readability: multi-line string.

```suggestion
       console.log(`
                                   ????                 ((((
                                ????????  (((        ((((((((
                            ???????????? (((((((  ((((((((((((
                        ????????#??####?(((((((((((((((((((((((
                    ####################((((((((((((((((((((((((
                ##########################################(((((((
            ##        (###       ####??   ##            #####  (((
        ######   #########   ###   ????   #######   ########/   ###
    ##########   #########   #???   ???   #######   ########     ###
 (((((((((((((        ((((   ????   ???   #######   #######   #   ###((#
    ((((((((((   ((((((???   ?(    ????   ???????   ??????   ???   ???#####
        ((((((   (((??????      ???????   ???????   ?????           ???#######
           (((        #???   ??????????   ???????   ????   ???????   ???#####
             ??????????????&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&??????
          ???? ??????  ?? && & &  &  &&? & & ?& && &  && & ???? ??   ?????
       ???????????????&&&&&&&&&&&&&&&&&&&&&&&&&&???????????????????????????
                       ???????????????(((
                      ?????? ((((((((((((
                                 ((((((((
                                    ((((((
                                        ((
`)
```